README.md

```diff
@@ -87,8 +100,30 @@ In order to run tests:
 go test -v ./...
 ```
 
+## Metrics
+
+Each service outputs various metrics to VictoriaMetrics, which is configured as Docker Compose container. These include business related metrics, latency (e.g. for database and API endpoints) metrics and other data consistency mechanism (like saga) metrics.
+
+Several ideas for alerts using metrics pushed to VictoriaMetrics and logging are:
+
+1. `currency-rate` service
+
+    1. Basically, any warn or error level log appearing in the log file.
```

Comment:
how would you implement this using victoria metrics? You'll need some kind of centralized logging solution, to analyze logs. Correct me if I'm wrong, please